README.md

```diff
@@ -1 +1,76 @@
-# modelplace
\ No newline at end of file
+This is an instruction on how to integrate a model to [Modelplace.AI](https://modelplace.ai/).  
+We support [PyTorch](https://pytorch.org/), [Tensorflow](https://www.tensorflow.org/), [OpenVINO](https://docs.openvinotoolkit.org/latest/index.html), and [OnnxRuntime](https://www.onnxruntime.ai/) backends. 
+
+## Publishing information
+For publishing you must provide information about yourself and the model
+### Required
+**Model description** - this is the whole description of what the model does (e.g. [model page](https://modelplace.ai/models/2) - the text in Summary block).
+
+**Model description preview short** - the text people see on a card on the [main page](https://modelplace.ai/) (just after the model name)
+
+**Model mame full** - on a model page - (e.g. Faster region-based convolutional neural network with ResNet-50 FPN backbone https://modelplace.ai/models/2)
+
+**Model name short** - on the [main page](https://modelplace.ai/) (e.g. Faster R-CNN).
+
+**Dataset name** - it's may be public data e.g. MSCOCO or Open images, or your internal dataset.
+
+**Preview image** - we will run a model on the image and will use its output for visualization. This image will be both on the [main page](https://modelplace.ai/) and on the [model page](https://modelplace.ai/models/2)
+
+**Licence** - licence type e.g. Apache 2.0, MIT, Proprietary and so on
+**Number of classes** 
+**Author** - your name or company name
+**Metrics** - accuracy of your model
+
+### Optional
+**Dataset link** - link to dataset that you use for training and testing of your model
+**Source link** - link to your project page or profile page or company page and so on
+**System requirements** - what system setup( e.g. number of CPU GPU RAM) we should use to serve your model.
+
+**Logo** - we will add it to the preview.   
+_Note: This should be svg(png is possible) image with circle shape and size 128x128 pixels at least._
+
+**Avatar image** - you can add your own logo(by default we use our).   
+_Note: This should be svg(png is possible) image with circle shape and size 128x128 pixels at least._
+
+## Evironment
+- Install venv  
+```python3.7 -m pip install virtualenv```
+- Create an empty virtual environment for python3.7  
+```python3.7 -m  virtualenv  venv```
+- Activate it  
+```source venv/bin/activate```
+- Install pytest and wheel  
+```python3.7 -m pip install pytest wheel```
+- Install git and git-lfs  
+```sudo apt install git git-lfs```
+- Clone the repo 
+```git clone https://github.com/opencv-ai/modelplace.git```
+
+## Example
+You can run [an example](pytorch_fastercnn) of a package
+- Go to package folder pytorch_fastercnn
+```cd pytorch_fastercnn```
+- Install the package  
+```python3.7 setup.py install develop```
```

Comment:
**Critical:**

That won't be working because it's not possible to install from git repository directly, pip must be used.

So the command must be:
`python3.7 setup.py bdist_wheel && rm -R build/ *.egg-info && pip3 install dist/*.whl`

Logs for `python3.7 setup.py install develop` if interested:
```
Searching for modelplace-api@ https://github.com/opencv-ai/modelplace-api/archive/v0.4.12.zip
Reading https://pypi.org/simple/modelplace-api/
Couldn't find index page for 'modelplace-api' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.org/simple/
No local packages or working download links found for modelplace-api@ https://github.com/opencv-ai/modelplace-api/archive/v0.4.12.zip
error: Could not find suitable distribution for Requirement.parse('modelplace-api@ https://github.com/opencv-ai/modelplace-api/archive/v0.4.12.zip')
```